fhd_core/source_modeling/source_dft_multi.pro

```diff
@@ -129,13 +136,8 @@ IF Keyword_Set(n_spectral) THEN BEGIN
     
     IF Keyword_Set(dft_threshold) THEN BEGIN
         model_uv_arr=Ptrarr(n_pol,n_spectral+1)
-        if ~keyword_set(gaussian_source_models) then begin
-            edge_i = where((x_vec LT dft_edge_pix) OR (y_vec LT dft_edge_pix) OR (dimension-x_vec LT dft_edge_pix) OR (elements-y_vec LT dft_edge_pix),$
-                n_edge_pix, complement=center_pix_i, ncomplement=n_center_pix)
-        endif else begin
-            edge_i = where((x_vec LT dft_edge_pix) OR (y_vec LT dft_edge_pix) OR (dimension-x_vec LT dft_edge_pix) OR (elements-y_vec LT dft_edge_pix) OR $
-                gaussian_x,n_edge_pix, complement=center_pix_i, ncomplement=n_center_pix)
-        endelse
+        edge_i = where((x_vec LT dft_edge_pix) OR (y_vec LT dft_edge_pix) OR (dimension-x_vec LT dft_edge_pix) OR (elements-y_vec LT dft_edge_pix) OR $
+            gaussian_x,n_edge_pix, complement=center_pix_i, ncomplement=n_center_pix)
```

Comment:
I'm sure I'm missing something here, but why does this only tests `gaussian_x` and not also `gaussian_y`?

Reply:
I'm only testing for the shape in the x direction because it would be unphysical to have an infinitely flat cigar-shaped source. I can add the gaussian_y logic if you think it's worth it (it's just bulky, that's really the only reason against)

Reply:
OK. Please add a code comment explaining that choice.